client.py

```diff
@@ -58,38 +99,23 @@ def filter_commit_data(commit):
     for item in commit:
         each_commit = {}
         author_info = {}
+        commit_message = item['commit']['message']
+        message = re.sub('[*\n\r]', ' ', commit_message)
```

Comment:
You have forgotten to import re
As I can see in the changeset, you don't have it.

Reply:
Nice use of re by the way!